src/EhrgoHealth.WebService.UnitTests/TestPatient.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Xunit;
+using Newtonsoft.Json.Linq;
+using System.IO;
+
+/*
+This Xunit tests demonstrates Patient.JSON is well formed.
+I believe this allows us to be confident in the Patient.JSON when referring to
+it for example data.
+
+We may not manually parse the JSON anymore thanks to FHIR.DSTU21 package
+but at least it is controlled data and may come in handy if we run into any limitations
+with the FHIR.DSTU21 API.
+
+If I extend this test class, it will be with the FHIR.DSTU21 objects
+*/
+
+namespace EhrgoHealth.WebService.UnitTests
+{
+    public class TestPatient
+    {
+        JObject patientJSON;
+
+        //XUnit uses constructor for [SetUp]
+        public TestPatient()
+        {
+            patientJSON = JObject.Parse(File.ReadAllText(@"..\..\Patient.json"));
+        }
+
+        //Parses well-formed JSON for the gender of the patient.
+        [Fact]
+        public void TestReadPatientGender()
+        {
+            Assert.Equal("male", patientJSON.GetValue("gender"));

```

Comment:
Okay, this was an older unit test meant to verify that I could parse the JSON, because the JSON was well formed. It was meant to build confidence that JSON text file was valid example data and could serve as example code during implementation.  It was meant to fail if the JSON file was modified by accident. I'll remove it.